code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -571,7 +571,7 @@
 
 /datum/chemical_reaction/corgium/on_reaction(datum/reagents/holder, datum/equilibrium/reaction, created_volume)
 	var/location = get_turf(holder.my_atom)
-	for(var/i in rand(1, created_volume) to created_volume) // More lulz.
+	for(var/i in 1 to rand(1, created_volume) to created_volume) // More lulz.
```

Comment:
Well all of these are functioning, they're not broken, it's just a little obtuse

Reply:
FUCK!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!